Dockerfile.dev

```diff
@@ -1,9 +1,13 @@
 # Development Dockerfile with optimized hot reloading
-FROM node:24-alpine
+FROM node:24-bullseye
 
 # Install dependencies for better compatibility and PostgreSQL client
-RUN apk add --no-cache python3 make g++ postgresql-client
-RUN apk add vim
+RUN apt-get update \
+	&& apt-get install -y python3 make g++ postgresql-client vim 
+
+# Install poppler-utils and poppler-data for node-poppler
+RUN apt-get install -y poppler-data poppler-utils \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
Could you add this dependency to Dockerfile.prod?

Reply:
OK. I'll change it. but please revert temporarily if any unexpected issues occur in production.